mass_merge/hooks.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
What happens in this init hook is more properly done by a data xml, or even better as demo data.